### DISCUSSION TITLE
Room name and username in the url address?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#419] [General]
Hello, I saw once a example for networked-aframe where the room name and username can be placed in the address bar. Do anyone know where I can find this example at please?

Joseph

**Comment by vincentfretin** [comment]
Hello, I'm not aware of this example. If you want to change the title of the page, you can set `document.title`, see [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Document/title).

**Reply by Therealjosephchrzempiec** [reply]
Hello, Somene made an example to where it can change the username and room name in the titlebar. I remember seeing it but it was a long time ago.
